File: Android/core/src/com/pps/simulation/Space_consts.java
package com.pps.simulation;

public class Space_consts {
    public static final double TRIGONOMETRY_CONST = Math.sqrt(2) / 3;
    public static final int VECTORS_MODULO = 12;
    public static double[] velocity_probabilities = new double[]{0.97,0.03/6, 0.03/6, 0.03/6, 0.03/6, 0.03/6, 0.03/6};
    public static final int BOUND_SIZE = 25;

    public static double[][] X_vectors = new double[VECTORS_MODULO][2];

    static {
        X_vectors[0] = new double[]{1, 0};                        // 0 grade`s
        X_vectors[1] = new double[]{TRIGONOMETRY_CONST, 0.5};     // 30 grade`s
        X_vectors[2] = new double[]{0.5, TRIGONOMETRY_CONST};     // 60 grade`s
        X_vectors[3] = new double[]{0, 1};                        // 90 grade`s
        X_vectors[4] = new double[]{-0.5, TRIGONOMETRY_CONST};    // 120 grade`s
        X_vectors[5] = new double[]{-1 * TRIGONOMETRY_CONST, 0.5};  // 150 grade`s
        X_vectors[6] = new double[]{-1, 0};                       // 180 grade`s
        X_vectors[7] = new double[]{-1 * TRIGONOMETRY_CONST, -0.5}; // 210 grade`s
        X_vectors[8] = new double[]{-0.5, -1 * TRIGONOMETRY_CONST}; // 240 grade`s
        X_vectors[9] = new double[]{0, -1};                       // 270 grade`s
        X_vectors[10] = new double[]{0.5, -1 * TRIGONOMETRY_CONST}; // 300 grade`s
        X_vectors[11] = new double[]{TRIGONOMETRY_CONST, -0.5};   // 330 grade`s}
    }
}

File: PC/src/space_consts.py
import math

class Space_consts:
	trigonometry_const = math.sqrt(3)/2

	X0_vector = (1, 0)                               # 0   grade`s
	X1_vector = (trigonometry_const, 0.5)            # 30  grade`s
	X2_vector = (0.5, trigonometry_const)            # 60  grade`s
	X3_vector = (0, 1)                               # 90  grade`s
	X4_vector = (-0.5, trigonometry_const)           # 120 grade`s
	X5_vector = (-1*trigonometry_const, 0.5)         # 150 grade`s
	X6_vector = (-1, 0)                              # 180 grade`s
	X7_vector = (-1*trigonometry_const, -0.5)        # 210 grade`s
	X8_vector = (-0.5, -1*trigonometry_const)        # 240 grade`s
	X9_vector = (0, -1)                              # 270 grade`s
	X10_vector = (0.5, -1*trigonometry_const)        # 300 grade`s
	X11_vector = (trigonometry_const, -0.5)          # 330 grade`s

	X_vectors = (X0_vector, X1_vector, X2_vector, X3_vector,
		X4_vector, X5_vector, X6_vector, X7_vector, X8_vector,
		X9_vector, X10_vector, X11_vector)

	VECTORS_MODULO = len(X_vectors)  # 12 vector`s from 0 to 11
	BOUND_SIZE = 5

	# 0.95 - the probability of maintaining the direction, 0.03/6 - the probability of a change of direction
	velocity_probabilities = [0.97, 0.03/6, 0.03/6, 0.03/6, 0.03/6, 0.03/6, 0.03/6]